apps/web/pages/apps/installation/[[...step]].tsx

```diff
@@ -155,47 +159,46 @@ const OnboardingPage = ({
   });
 
   const handleSelectAccount = async (teamId?: number) => {
-    try {
-      setIsSelectingAccount(true);
-      if (appMetadata.isOAuth) {
-        const state = JSON.stringify({
-          appOnboardingRedirectUrl: getAppOnboardingRedirectUrl(appMetadata.slug, teamId),
-          teamId,
-        });
-
-        const res = await fetch(
-          `/api/integrations/${
-            appMetadata.slug == "stripe" ? "stripepayment" : appMetadata.slug
-          }/add?state=${state}`,
-          {
-            method: "GET",
-            headers: {
-              "Content-Type": "application/json",
-            },
+    mutation.mutate(
+      {
+        isOmniInstall: true,
```

Comment:
setting `isOmniInstall` to true so that we can be redirected back to /apps/installation/[step]?slug
if not set it redirects to `apps/installed/[category]` and the redirection can't be controlled